core/objects.go

```diff
@@ -347,6 +348,13 @@ func (ch Challenge) checkConsistency() error {
 	return nil
 }
 
+func (ch Challenge) StringID() string {
```

Comment:
Can you add a comment that describes this function? It's not easy to deduce the higher level purpose from the code alone.

Reply:
Done.